mix.exs

```diff
@@ -4,7 +4,7 @@ defmodule OpentelemetryTesla.MixProject do
   def project do
     [
       app: :opentelemetry_tesla,
-      version: "1.0.0-rc.1",
+      version: "1.1.0-rc.1",
```

Comment:
as it was not stable and it's not backwards-incompatible change, I'd go with 0.2.0-rc.1

Reply:
Actually, I think I didn't nailed it from the beginning. Even though I agree with your comment isn't a bit weird going from a `1.0.0` to a `0.2.0`?

Reply:
oh nvm. I thought the previous version was 0.1. Going from 1.0 to 1.1 is totally fine here.